test/support/factory.ex

```diff
@@ -51,6 +51,33 @@ defmodule Wanda.Factory do
     }
   end
 
+  def env_factory(attrs) do
```

Comment:
If I am understanding it correctly `env_event_factory` is used in tests _closer_ to the integration layer where `ExecutionRequested` event is involved.
While `env_factory` is used in more inner layers.

What other differences are there between `env_factory` `env_event_factory`?
Would a single factory be enough?

Reply:
`env_factory` creates data based in our domain (where the env is simply a map).
`env_event_factory` creates the env dataset coming in the `ExecutionRequested` env field.
They are different, as the 2nd is strictly attached to protobuf and it is only used to send and receive messages, not internally.

Maybe I could rename to `execution_requested_env_factory`